product_docs/docs/tde/15/index.mdx

```diff
@@ -75,7 +75,9 @@ Data encryption and decryption is managed by the database and doesn't require ap
 
 EDB Postgres Advanced Server and EDB Postgres Extended Server provide hooks to key management that's external to the database. These hooks allow for simple passphrase encrypt/decrypt or integration with enterprise key management solutions. See [Securing the data encryption key](./key_stores) for more information.
 
-### How does TDE encrypt data? 
+### How does TDE encrypt data?
+
+EDB TDE uses [OpenSSL](https://openssl-library.org/) to encrypt data files with the AES encryption algorithm. In Windows systems, TDE uses [OpenSSL 3](https://docs.openssl.org/3.0/). In Linux systems, TDE uses the OpenSSL version installed in the host operating system. To check the installed version, run `openssl version`. For more details, refer to the [OpenSSL documentation](https://docs.openssl.org/master/).
```

Comment:
LGTM

Reply:
I used the wording "OpenSSL that their operating system installs as a server package dependency" just because some customers will use a build of OpenSSL that does not come from the OpenSSL community e.g.. With the wording change mentioned here, I'd change the last sentence to "For more information, see the [OpenSSL documentation](https://docs.openssl.org/master). If you're using a custom build not provided by the OpenSSL community, consult your vendor's documentation."

Reply:
Makes sense. I made the changes ✅